### ISSUE TITLE
Make zoom controls button elements since they are buttons. Buttons are for actions, links are to take you to another page.

### ISSUE THREAD

**Issue by prushforth** [Maps4HTML/Web-Map-Custom-Element#19]
This was the suggestion of @powersurge on a11yslackers.  He is correct, and perhaps this issue should be raised over at the Leaflet.js project, since the zoom control comes from there.

**Comment by Malvoz** [comment]
To my understanding, anchors are perfectly valid as buttons in this instance because `role="button"` is used, the buttons have accessible labels, and they respond correctly to keyboard input. Although I'm not certain, I think the usage of `<a>` is because buttons and other form elements are generally harder to style, e.g. they don't inherit font styles by default, and CSS [appearance](https://developer.mozilla.org/en-US/docs/Web/CSS/appearance) doesn't have (or didn't have at the time) wide enough browser support to warrant its use.

Anyhow, there's a leaflet issue to tackle some accessibility issues in https://github.com/Leaflet/Leaflet/issues/3210, including moving from links to buttons.

**Comment by prushforth** [comment]
I think that's good enough for me at this time. Closing; feel free to discuss if necessary.

**Comment by Malvoz** [comment]
https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/19#issuecomment-578974225:
> they respond correctly to keyboard input

That statement is incorrect, see upstream issue https://github.com/Leaflet/Leaflet/issues/7369.

**Comment by Malvoz** [comment]
This issue should consider not only Zoom controls but all standard Leaflet controls, such as a popup's Close button, Fullscreen control, Layer control.
